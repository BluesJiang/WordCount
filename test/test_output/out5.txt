test\test_cases\httpd.c, 字符数: 16221
test\test_cases\httpd.c, 单词数: 1757
test\test_cases\httpd.c, 行数: 516
