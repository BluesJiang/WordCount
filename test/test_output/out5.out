test\test_cases\httpd.c, 字符数: 16221
test\test_cases\httpd.c, 单词数: 1757
test\test_cases\httpd.c, 行数: 516
